ruby/.rubocop.yml

```diff
@@ -6,6 +6,9 @@ AllCops:
     - db/schema.rb
     - bin/**/*
 
+inherit_from:
+  - rubocop/.style.yml
```

Comment:
I was wondering if inheriting from a local file here will work in the BuoyRails codebase where we download files like so:

```
inherit_from:
  - https://raw.githubusercontent.com/BuoySoftware/guides/main/rails/.rubocop.yml
  - https://raw.githubusercontent.com/BuoySoftware/guides/main/ruby/.rubocop.yml
```

Just something to keep an eye on, hopefully it'll work!

Reply:
Good question! I think you are correct. Lemme try pulling from this PR directly to see what happens. I believe that Rubocop will only cache the directly referenced remote url so the inherit_from I have here won't work as expected

Reply:
Ok, well surprising result here. I tried this out locally in BuoyRails by changing the inherit_from to the following:

```
inherit_from:
  - https://raw.githubusercontent.com/BuoySoftware/guides/main/rails/.rubocop.yml
  - https://raw.githubusercontent.com/BuoySoftware/guides/a0ce29711104dc3e2b64d212b157bc3d40802d8f/ruby/.rubocop.yml
```

It worked correctly by changing `super` from failing linting check to passing. So I guess we are good here?

Reply:
Nice, I guess Rubocop is smart enough to reuse the the Github URL!